package.json

```
@@ -135,7 +135,7 @@
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/preset-env": "7.22.9",
     "@babel/preset-typescript": "7.22.5",
-    "@babel/runtime": "7.22.6",
+    "@babel/runtime": "^7.22.15",
```

Please fix the package version
```suggestion
    "@babel/runtime": "7.22.15",
```